clojure-skeleton/src/ant_battles/symbolizer.clj

```diff
@@ -0,0 +1,9 @@
+(ns ant-battles.symbolizer)
+
+(defn symbolize [arg]
```

Comment:
This name is misleading because in Clojure `symbols` and `keywords` are different things. This function is generating keywords not symbols.

Another potential issue with this is generating a bunch of non-garbage-collected keywords from untrusted incoming requests. This is generally advised against because someone could hammer the API with randomly generated json keys, eating up memory until the server crashes. In practise it shouldn't be a problem for the ant server and I guess it doesn't need to be Production Ready™.